xml/System.Web.Security/SqlMembershipProvider.xml

```diff
@@ -1259,7 +1259,7 @@
   
 -or-
   
- The <see langword="passwordFormat" /> attribute is set to <see langword="Encrypted" /> and the  configuration element specifies <see langword="AutoGenerate" /> for the <see langword="decryptionKey" /> attribute.  
+ The <see langword="passwordFormat" /> attribute is set to <see langword="Encrypted" /> and the <code>&lt;machineKey&gt;</code> configuration element specifies <see langword="AutoGenerate" /> for the <see langword="decryptionKey" /> attribute.  
```

Comment:
machineKey element: <https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/w8h3skw9(v%3dvs.100)>. This would have to be in \<see href=""> format.